src/config/featured.json

```diff
@@ -14,6 +14,13 @@
       "description": "KNIX MicroFunctions is a serverless computing platform that combines container-based resource isolation with a lightweight execution model using processes to significantly improve resource efficiency and decrease the function startup latency. KNIX MicroFunctions works in Knative as well as bare metal or virtual machine-based environments.",
       "logo": "https://raw.githubusercontent.com/knix-microfunctions/knix/develop/logo/KNIX_logo_stack.jpg"
     },
+    {
+      "name": "NTT",
+      "repo_url": "https://github.com/nokia/ntt",
+      "homepage": null,
```

Comment:
```suggestion
      "homepage": "https://nokia.github.io/ntt/",
```

Reply:
Repo url will be already a clickable link.

Reply:
This is not the Repo, but the homepage. Would be nice if we'd link this.

Reply:
Ah okay, now I realized, that this is different from the repo link.